### DISCUSSION TITLE
Correspondence Materials-zero energy control

### DISCUSSION THREAD

**Discussion by guangdongzhang** [peridigm/peridigm#270] [Q&A]
Hi everyone,
     The question I would like to know is how Peridigm  controls the zero-energy mode for Correspondence Materials. Or there is no zero-energy mode control in the code? Thank you very much.

**Comment by djlittl** [comment]
Peridigm uses the method in:

Littlewood, David J. "A nonlocal approach to modeling crack nucleation in AA 7075-T651." ASME International Mechanical Engineering Congress and Exposition. Vol. 54945. 2011.

See page 4 of that paper.

<img width="1100" alt="image" src="https://user-images.githubusercontent.com/5599237/191078681-bc748432-c330-494f-a865-fdfa7586ca4d.png">

**Reply by guangdongzhang** [reply]
Okay, thank you.
